core/templates/services/UpgradedServices.ts

```diff
@@ -531,6 +559,8 @@ export class UpgradedServices {
     upgradedServices['ChangeObjectFactory'] = new ChangeObjectFactory();
     upgradedServices['ClassifierObjectFactory'] = new ClassifierObjectFactory();
     upgradedServices['CodeNormalizerService'] = new CodeNormalizerService();
+    upgradedServices['ComputationDataObjectFactory'] =
```

Comment:
Add these too in sorted order (here and everywhere in this file).

Reply:
done